keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,91 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
+    """Pads the given bounding box tensor with -1s.
+
+       This is done to convert RaggedTensors to be converted into
+       standard Dense tensors, which have better performance and
+       compatibility within the TensorFlow ecosystem.
+
+    Args:
+      bounding_boxes: a ragged tensor of bounding boxes.
+      Can be batched or unbatched.
+
+    Returns:
+      bounding_boxes_tensor: a tensor containing the -1 padded bounding boxes.
+    """
+    return bounding_boxes.to_tensor(-1)
+
+
+def filter_sentinels(bounding_boxes):
+    """converts a Dense padded bounding box `Tensor` to a `tf.RaggedTensor`.
+
+    Args:
+      bounding_boxes: a tensor of bounding boxes.
+      Can be batched or unbatched.
+
+    Returns:
+      bounding_boxes_tensor: a ragged tensor containing filtered bounding boxes.
```

Comment:
you can drop the return name and just say: "a ragged Tensor containing the filtered bounding boxes"